travel.py

```diff
@@ -214,80 +424,253 @@ def signup(tenant):
+    @cross_origin(supports_credentials=True)
+    def updateflights(tenant, username):
+        """Book a new flight for a user
+        ---
+        tags:
+        - tenants
+        parameters:
+            - name: tenant
+              in: path
+              required: true
+              schema:
+                type: string
+              example: tenant_agent_00
+              description: Tenant agent name
+            - name: username
+              in: path
+              required: true
+              schema:
+                type: string
+              example: user1
+              description: Username
+        requestBody:
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    flights:
+                      type: array
+                      format: string
+                      example: [{
+                                   "name": "boeing",
+                                   "flight": "12RF",
+                                   "price": 50.0,
+                                   "date": "12/12/2020",
+                                   "sourceairport": "London (Gatwick)",
+                                   "destinationairport": "Leonardo Da Vinci International Airport"
+                               }]
+        responses:
+            200:
+              description: Returns login data and query context information
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      context:
+                        type: array
+                        format: string
+                        example: ["A description of a KV operation"]
+                      data:
+                        type: object
+                        properties:
+                          added:
+                            type: object 
+                            example: {
+                                        "date": "12/12/2020",
+                                        "destinationairport": "Leonardo Da Vinci International Airport",
+                                        "flight": "12RF",
+                                        "name": "boeing",
+                                        "price": 50.0,
+                                        "sourceairport": "London (Gatwick)"
+                                     }
+            401:
+              description: Returns an authentication error
+              content:
+                application/json:
+                    schema:
+                       type: object
+                       properties:
+                        message:
+                           type: string
+                           example: ["User does not exist"]
+        security:
+            - bearer: []
+        """
+        agent = make_user_key(tenant)
+        user = make_user_key(username)
 
```

Comment:
thanks Git, for assuming that these blank lines are useful context and completely misinterpreting the diff... *sigh*